xbmc/music/Album.cpp

```diff
@@ -85,8 +84,8 @@ CAlbum::CAlbum(const CFileItem& item)
         }
       }
 
-      if (artistName.empty() && tag.GetMusicBrainzAlbumArtistID().size() == artist.size())
-        artistName = artist[i];
+      if (artistName.empty() && tag.GetMusicBrainzAlbumArtistID().size() == tag.GetArtist().size())
+        artistName = tag.GetArtist()[i];
```

Comment:
I think this needs to be changed to tag.GetAlbumArtist() instead of tag.GetArtist() for this line and the previous line.